### PR TITLE
Fresh fruits

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -39205,11 +39205,11 @@ zh:水果
...
 
 <en:Fruits
-en:French fruits
+en:Fresh fruits
+fr:Fruits frais
 
 <en:Fruits
-fr:Cassis frais
-season_in_country_fr:en: 6,7,8
+en:French fruits
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39212]: Why do we have a "French fruits" category?

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -39432,12 +39448,10 @@ ciqual_food_name:en:Black mulberry, raw
 ciqual_food_name:fr:Mûre noire (du mûrier), crue
 
 <en:Blackberries
-fr:Mûres fraîches
+<en:Fresh fruits, Raw blackberries
+en:Fresh blackberries
+fr:Mûres fraîches, Mûres de ronce crues
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39453]: Shouldn't "Mûres de ronce crues" be a synonym of the parrent "Raw blackberries"? Or do we assume "Mûres de ronces crues" are never frozen?

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -39463,12 +39477,14 @@ ciqual_food_name:en:Blueberry, raw
...
 <en:Blueberries
+<en:Fresh fruits
+en:Fresh blueberries
 fr:Myrtilles fraîches
 season_in_country_fr:en: 7,8,9
 
 <en:Blueberries
-en:Frozen raw Blueberry
-fr:Myrtille surgelée crue
+en:Frozen raw bluberries, Frozen raw blueberry
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39486]: I'm wondering if we should put "Frozen blueberries" as the main name for "Frozen raw blueberries".  (also there'a missing in "bluberries")

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -39665,6 +39683,8 @@ ro:Lămâi
 ru:Лимоны
 
 <en:Lemons
+<en:Fresh fruits
+en:Fresh limons
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39687]: lemons

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -39944,6 +39967,8 @@ fr:Kiwis
 nl:Kiwis
 
 <en:Kiwifruits
+<en:Fresh fruits
+en:Fresh kiwifruits
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:39971]: + Fresh kiwis as a synonym

File: taxonomies/categories.txt (hunk 3 above)
Comment (stephanegigandet) [reply taxonomies/categories.txt:39486]: ok just saw below that we have another "Frozen blueberries" category. I think we should merge them.

File: taxonomies/categories.txt (hunk 1 above)
Comment (moon-rabbitOFF) [reply taxonomies/categories.txt:39212]: Pierre created it for the DOP fruits I think

**Comment by stephanegigandet** [comment]
There a few small issues that prevent the taxonomy to build, I'm fixing them.

**Pull request merged**